README.md

```diff
@@ -1118,6 +1158,8 @@ the disk itself while you were cutting out the new notch...
 
 ####  01AC Do you use the "P convention"?
 
+Common Lisp and some other languages use the "p" suffix for predicates, e.g. `(numberp 42)`. Newer lisps like Scheme and Clojure usually go with more intuitive "?" instead, like `(string? "foo")`.
```

Comment:
See also the Jargon File: http://catb.org/jargon/html/p-convention.html

Reply:
Added.